textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/regexmatch/RegexToGramQueryTranslatorTest.java

```diff
@@ -0,0 +1,29 @@
+package edu.uci.ics.textdb.dataflow.regexmatch;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * @author Shuying Lai
+ * @author Zuozhi Wang
+ */
+
+public class RegexToGramQueryTranslatorTest {
```

Comment:
Test cases for equals are in GramBooleanQueryTest.java. This RegexToGramQueryTranslatorTest.java is testing translator.